pages/portfolio.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/common.css">
+  <link rel="stylesheet" href="../styles/portfolio.css">
+  <title>Portfolio</title>
 </head>
-<body>
-  
-</body>
+
+<div class="wrapper">
+  <header class="box header">
+    <h1>My Portfolio</h1>
+  </header>
+
+  <nav class="box sidebar">
+    <ul class="no-bullets center">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about.html">About</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+    </ul>
+  </nav>
+
+  <body class="box content">
```

Comment:
Same issue with the body tag here. All of the non-head content should be in body. We could simply remove the extra body tags here then.